src/Test.UnitTests.Sarif.Driver/Sdk/AnalyzeCommandBaseTests.cs

```diff
@@ -663,11 +663,100 @@ public void AnalyzeCommandBase_ReportsWarningOnUnsupportedPlatformForRuleAndNoRu
             return run;
         }
 
+        [Fact]
+        public void AnalyzeCommand_Traces()
```

Comment:
>AnalyzeCommand_Traces [](http://example.com/codeflow?start=20&length=21)

Good things about this test: there are minimal literal constant 'magic values', it runs a comprehensive set of tests and will report on all break (instead of failing on the first one), it nicely buckets all trace testing in one place and can easily be extended. This change also makes some nice incremental improvements to the general test methodology, for example, returning the Run object from the test helper to assist in validation.